R/api-bump-version.R

```diff
@@ -16,8 +20,11 @@
 #' Fear not, run [unbump_version()], merge that PR, run `bump_version()`.
 #'
 #' @example man/examples/bump-version.R
-bump_version <- function(which = "dev") {
+bump_version <- function(which = "dev", no_change_behavior = c("bump", "noop", "fail")) {
   check_which(which)
+
+  no_change_behavior <- match.arg(no_change_behavior, choices = c("bump", "noop", "fail"))
```

Comment:
`rlang::arg_match()` gives rlang-style error messages, can we pick up the options from the formals?


```suggestion
  no_change_behavior <- arg_match(no_change_behavior)
```